native/common/include/jp_proxy.h

```diff
@@ -39,10 +39,14 @@ class JPProxy
 		return m_Context;
 	}
 
-	virtual JPPyObject getCallable(const string& cname) = 0;
+	virtual JPPyObject getCallable(const string& cname, bool& addSelf) = 0;
```

Comment:
passing bool by reference seems uncommon, but works anyway. Feel free to change it or not.